assets/app/view/game/spreadsheet.rb

```diff
@@ -77,11 +79,17 @@ def render_player_history
           end
           next if values == last_values
 
+          rel_values = (last_values || Array.new(values.length, 0)).map(&:-@).zip(values).map(&:sum) if @rel_player_hist
           last_values = values
           zebra_row = !zebra_row
           h(:tr, zebra_props(zebra_row), [
             h('th.left', h.round),
-            *values.map { |v| h('td.padded_number', @game.format_currency(v)) },
+            *values.map.with_index do |v, i|
```

Comment:
i'm not a fan of embedding this inside the array, can you move this out and just call it a var 

xxx = values.map...